src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -176,6 +213,14 @@ class ReportActionItemMessageEdit extends React.Component {
                         selection={this.state.selection}
                         onSelectionChange={this.onSelectionChange}
                     />
+                    <View style={styles.editChatItemEmojiWrapper}>
+                        <EmojiPickerButton
+                            isDisabled={shouldDisableEmojiPicker}
+                            onModalHide={() => InteractionManager.runAfterInteractions(() => this.textInput.focus())}
```

Comment:
Coming from https://github.com/Expensify/App/issues/30119

If, at the beginning, a composer's emoji picker is open, opening some _other_ emoji picker results in a race:
- Here, we focus the composer because the original emoji picker hides
- The other emoji picker focuses its search field